events/index.html

```diff
@@ -0,0 +1,58 @@
+---
+pagename: Foreman Events
+layout: event_index
+---
+{% assign events = site.data.events | filter_old_events %}
+
+<div class='col-md-12'>
+  <div class='col-md-6 pull-left'>
+    <p>Here you'll find all the Foreman-related events where members of the
+    community are talking about their use of, or contribution to, Foreman.
+    Attending one of these events is a great way to learn more about Foreman!<p>
+    <p>You can <a href='/events/all.ics'>subscribe to these events</a> via our ICS feed!</p>
+    <p>Know of an event not listed here? .. or want to fix some details? Please
+    <a href='https://github.com/theforeman/theforeman.org/blob/gh-pages/_data/events.yaml'>contribute event info</a>!</p>
+  </div>
+  <div class='col-md-6 col-offset-6 pull-right'>
+    <div id='calendar'></div>
+  </div>
+</div>
+
+{% for event in events %}
+<div class="blog-item post-content" id="{{ event.title | slugify }}">
```

Comment:
Added 20pm to the base of the calendar - adding to the top of the text list just didn't seem to be working